.drone.yml

```diff
@@ -31,9 +31,9 @@ pipeline:
     environment:
       - DOCKER_HOST=tcp://172.17.0.1:2375
     commands:
-      - docker login -u="ukhomeofficedigital+drone" -p=${DOCKER_PASSWORD} quay.io
-      - docker tag file-vault-${DRONE_COMMIT_SHA} quay.io/ukhomeofficedigital/file-vault:${DRONE_COMMIT_SHA}
-      - docker push quay.io/ukhomeofficedigital/file-vault:${DRONE_COMMIT_SHA}
+      - docker login -u="ukhomeofficedigital+drone" -p=$${DOCKER_PASSWORD} quay.io
+      - docker tag file-vault-${DRONE_COMMIT_SHA} quay.io/ukhomeofficedigital/file-vault:$${DRONE_COMMIT_SHA}
```

Comment:
You have `$` in one instance of `DRONE_COMMIT_SHA` on this line and `$$` in the other. I imagine it should be consistent.